### PR TITLE
Add decoder for degen airdrop

### PR DESCRIPTION
**Pull request by yabirgb**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

**Comment by yabirgb** [comment]
In the two failed tests the reason was a remote error. One didn't manage to connect to the rpc node and the other had a read timeout on etherscan

File: rotkehlchen/chain/base/modules/degen/decoder.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            token_amount=raw_amount,
+            token_decimals=DEFAULT_TOKEN_DECIMALS,
+        )
+
+        for event in context.decoded_events:
+            if event.event_type == HistoryEventType.RECEIVE and event.location_label == user_address and amount == event.balance.amount and self.degen == event.asset:  # noqa: E501
+                event.event_type = HistoryEventType.RECEIVE
+                event.event_subtype = HistoryEventSubType.AIRDROP
+                event.counterparty = CPT_DEGEN
+                event.notes = f'Claimed {amount} DEGEN from Degen airdrop 1'
```
Comment (LefterisJP) [inline rotkehlchen/chain/base/modules/degen/decoder.py:51]:
```suggestion
                event.notes = f'Claimed {amount} DEGEN from Degen airdrop 2'
```

File: rotkehlchen/chain/base/modules/degen/constants.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from typing import Final
+
+from rotkehlchen.chain.evm.types import string_to_evm_address
+
+
+CPT_DEGEN = 'degen'
+CLAIM_AIRDROP_1_CONTRACT: Final = string_to_evm_address('0x88d42b6DBc10D2494A0c6c189CeFC7573a6dCE62')  # noqa: E501
```
Comment (LefterisJP) [inline rotkehlchen/chain/base/modules/degen/constants.py:7]: This is airdrop 2

```suggestion
CLAIM_AIRDROP_2_CONTRACT: Final = string_to_evm_address('0x88d42b6DBc10D2494A0c6c189CeFC7573a6dCE62')  # noqa: E501
```

**Review by LefterisJP — COMMENTED** [review]
Add a short changelog entry too

**Review by LefterisJP — APPROVED** [review]
lgtm

**Pull request merged**